pkg/helper/client.go

```diff
@@ -15,15 +16,34 @@ import (
 
 const nmstateCommand = "nmstatectl"
 
-func nmstatectl(arguments ...string) (string, error) {
-	cmd := exec.Command(nmstateCommand, arguments...)
-	var outb, errb bytes.Buffer
-	cmd.Stdout = &outb
-	cmd.Stderr = &errb
+func show(arguments ...string) (string, error) {
+	cmd := exec.Command(nmstateCommand, "show")
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
 	if err := cmd.Run(); err != nil {
-		return "", fmt.Errorf("Failed to execute nmstatectl show: '%v'", err)
+		return "", fmt.Errorf("Failed to execute nmstatectl show: '%v', %s", err, stderr.String())
```

Comment:
errors should start lower-case. Would you mind fixing it here while we edit these?

Reply:
Done